sdk/spring/azure-spring-boot-test-aad-obo/pom.xml

```diff
@@ -49,6 +49,12 @@
       <artifactId>spring-boot-starter-test</artifactId>
```

Comment:
To make less confusing, how about rename this module to `azure-spring-boot-test-aad-webapp-and-resource-server-with-obo`?